packages/orbit-components/src/Illustration/index.tsx

```diff
@@ -5,8 +5,8 @@ import * as React from "react";
 import IllustrationPrimitive, { SIZE_OPTIONS } from "../primitives/IllustrationPrimitive";
 import type { Props } from "./types";
 
-const Illustration = ({ size = SIZE_OPTIONS.MEDIUM, ...props }: Props) => (
-  <IllustrationPrimitive {...props} size={size} />
+const Illustration = ({ size = SIZE_OPTIONS.MEDIUM, alt = "", role = "img", ...props }: Props) => (
```

Comment:
These two default values are incompatible. We can't have empty `alt` and `role` _img_. I would prefer to have `alt` _undefined_ and `role` _presentation_ by default. We can even make `alt` required if `role` is _img_…

Ofc it all needs proper documentation

Reply:
yeah, I was not sure about the combination. I'll go with `alt=undifined` && `role="presentation`. 👍 

> Ofc it all needs proper documentation 
I'll mention that in related doc-ticket. 👍